netutils/iperf/iperf.c

```diff
@@ -338,7 +338,7 @@ static void iperf_report_task(FAR void *arg)
              ts_diff(&start, &start),
              ts_diff(&now, &start),
              now_len,
-             ((double)(now_len * 8) / 1000000) /
+             ((double)(now_len * 8) / 1000000.0) /
              (double)ts_diff(&now, &start)
```

Comment:
please remove the cast for all ts_diff too

Reply:
Done